mlflow_faculty/trackingstore.py

```diff
@@ -292,5 +273,38 @@ def search_runs(self, experiment_ids, search_expressions, run_view_type):
         else:
             return [faculty_run_to_mlflow_run(run) for run in faculty_runs]
 
-    def log_batch(self, run_id, metrics, params, tags):
-        raise NotImplementedError()
+    def log_batch(self, run_uuid, metrics=None, params=None, tags=None):
+        """
+        Fetches the experiment by ID from the backend store.
+
+        :param experiment_id: Integer id for the experiment
```

Comment:
Remove this line?

Reply:
Between the description and the param list?